### PR TITLE
Add support for Huion MiniDial KD100

### PR DESCRIPTION
**Pull request by albfan**
- [ ] Ring is still not correctly configured ()
- [x] mouse button (was just another button)
- [x] This only covers KD100 connected by wire. 
- [ ] Wireless do not match usb filter (it only shows 256c:006d)
- [x] Only eight buttons are working (using GNOME). Fix rest of buttons in digimend driver

relates #469

### CODE REVIEW

**Comment by albfan** [comment]
Found how to detect third layer of buttons (9-12) https://gitlab.freedesktop.org/libinput/libinput/-/issues/730

**Comment by albfan** [comment]
Fourth and Fifth layer probably needs something on digimend driver

**Review by whot — APPROVED** [review]
Looking through this: the data file itself and SVG appear fine. Since libwacom doesn't control the actual functionality of the tablet, I think we're good to merge this?

**Comment by albfan** [comment]
buttons are correctly reported, fixing degimend driver

**Comment by albfan** [comment]
Mouse button is reported too as a pad button "ButtonS"

So all pad buttons are recognized not and shown in svg

File: data/huion-kd100.tablet [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# *---------*
+# |         |
+# |         |
+# |         |
+# *---------*
+#
+
+[Device]
+Name=Huion KD100
+DeviceMatch=usb:256c:006d:HUION Huion Tablet_KD100 Keyboard;usb:256c:006d:HUION Huion Tablet_KD100 Pad;usb:256c:006d:HUION Huion Tablet_KD100 Consumer;usb:256c:006d:HUION Huion Tablet_KD100 Pen;usb:256c:006d:HUION Huion Tablet_KD100
```
Comment (whot) [inline data/huion-kd100.tablet:38]: this isn't the right match, you don't want libwacom to match on all event nodes this device produces, only on the one that sends the actual events.

Comment (albfan) [reply data/huion-kd100.tablet]: Got it now

**Comment by whot** [comment]
AIUI this is good enough for the libwacom bits, the kernel drivers are separate so let's merge this!

**Pull request merged**